src/Modules/Admins/components/add_new_admin_button.tsx

```diff
@@ -0,0 +1,33 @@
+import { Box, Button } from "@chakra-ui/react";
```

Comment:
use pascalCase for components and tsx file names 
eg : AddNewAdminButton

Reply:
Okay